src/React.purs

```diff
@@ -372,11 +372,17 @@ createClass :: forall props state render eff.
   ReactSpec props state render eff -> ReactClass props
 createClass spc = runFn2 createClass' toNullable spc
 
--- | Create a stateless React class.
-createClassStateless :: forall props render.
+-- | Create a stateless React class.  When using a non anonymous function the
+-- | displayName will be the capitalized name of the function, e.g.
+-- | ``` purescript
+-- | helloWorld = createClassStatelesss hellowWorldCls
+-- |    where
+-- |      hellowWorldCls props = ...
+-- | ```
+-- | Then the `displayName` will be set up to `HellowWorldCls`
```

Comment:
issue #118 @ethul this might be a bit surprising but it's best of what we can get.